src/main/java/core/basesyntax/service/impl/ParserServiceImpl.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.dao.TransactionStorageDao;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.ParserService;
+import java.util.List;
+import java.util.Optional;
+
+public class ParserServiceImpl implements ParserService {
+    private static final int CODE_POSITION = 0;
+    private static final int FRUIT_POSITION = 1;
+    private static final int QUANTITY_POSITION = 2;
+    private static final String SEPARATOR = ",";
+
+    private TransactionStorageDao transactionStorageDao;
+
+    public ParserServiceImpl(TransactionStorageDao transactionStorageDao) {
+        this.transactionStorageDao = transactionStorageDao;
+    }
+
+    @Override
+    public void parse(List<String> rawData) {
+        for (String element : rawData) {
+            String[] parts = element.split(SEPARATOR);
+            Optional<FruitTransaction.Operation> operation = FruitTransaction.Operation
+                    .fromString(parts[CODE_POSITION]);
+            if (operation.isEmpty()) {
+                continue;
+            }
```

Comment:
I guess you made this logic for 1 line case. Try a different approach, that's not a good one.